lib/config/codec_switching_strategy.js

```diff
@@ -0,0 +1,26 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.provide('shaka.config.CodecSwitchingStrategy');
+
+/**
+ * @enum {string}
+ * @export
+ */
+shaka.config.CodecSwitchingStrategy = {
+  // No codec switching allowed (i.e. current Shaka behaviour).
```

Comment:
nit: "current" should say "default".  Also, you should move these comments into jsdoc syntax so that they end up in the public API docs for these values.